mosaic-site/src/components/components/ComponentsList.module.css

```diff
@@ -0,0 +1,68 @@
+.componentList {
+  --table-cell-padding: 0.75rem;
+}
+
+.componentList {
+  margin: calc(var(--salt-size-unit) * 3) 0;
+}
+
+.componentList table {
+  table-layout: fixed;
+}
+
+.componentList td :first-child {
+  display: flex;
+  align-items: baseline;
+  gap: calc(var(--salt-size-unit) / 2);
+}
+
+.componentList th {
+  padding: 0;
+}
+
+.componentList th button {
+  /* Remove button default styles so it's not visible but we retain its functionality for ADA purposes */
+  --saltButton-background-hover: transparent;
+  --saltButton-background: transparent;
+  --saltButton-background-active: transparent;
+  --button-text-color-active: var(--button-text-color-hover);
+  --saltButton-width: 100%;
+  --saltButton-padding: 0;
+  --saltButton-text-color: none;
+  --saltButton-textTransform: none;
+  --saltButton-fontWeight: none;
+  --saltButton-letterSpacing: normal;
+  --saltButton-height: auto;
+  --saltIcon-color: inherit;
+  --salt-actionable-primary-foreground-active: var(--button-text-color-hover);
```

Comment:
we shouldn't redefine any theme here. what's the purpose? there's probably a better way to get what's desired

Reply:
We added this in the current site because we want the table headers to be accessible by having a button to sort the content but it means we have to remove the button styles so it still looks like a normal table. Should these be defined elsewhere?

Reply:
i just committed something i believe works :) lmk if its okay

Reply:
It looks good :) thank you!